doc/components/extrema.md

```diff
@@ -0,0 +1,71 @@
+# Extrema
+
+ROHD-HCL provides a component to find the extrema of a list of Logic.
+
+The component `Extrema` will determine the extremas (maximum or minimum) and their position in a list of `Logic`s.
+
+## Description
+
+`Extrema` will take in an input containing a `List<Logic>` along with parameters for `max` and `first`.
+
+* `max` is a boolean indicating whether to find the maximum value (`true`), or the minimum value (`false`). Default is `true`.
+
+* `first` is a boolean indicating whether to find the first instance of the extrema (`true`), or the last instance (`false`). Default is `true`.
+
+`Extrema` will then ouput two `Logic` signals:
+
+* `index` The index or position of the extrema in the list
+  >`index` will be a `Logic` with the smallest width needed to represent the largest possible index in the list.
```

Comment:
i think you can just do `  *` (with some spaces before the `*`) to make an indented bullet.  the `>` is a quote